### PR TITLE
Add Title-IX related contact information

### PR DESCRIPTION
**Pull request by elijahverdoorn**
I was contacted by Kari Hohn, the college Title-IX Coordinator about putting some more information into the app. We've been emailing about it, and settled on what you see here. I'm starting to wonder if there is a need for a dedicated Resources section of the app, as important contacts seems to be a little cluttered (Pause Kitchen feels even more out of place now!). Maybe adding tabs to the Contacts section would fix the problem and let us add a few more in the future without it becoming a massive list?

@erichkauffman I'd encourage you to reach out to the It's On Us chairs for this upcoming year and see what they think about this data change.

### CODE REVIEW

**Comment by hawkrives** [comment]
I do like the idea of splitting them into tabs within the contacts screen. I'll go ahead and merge this, and we can open an issue about reorganizing them?

File: data/contact-info/5-college-pastors.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title: College Pastors
+phoneNumber: '5077863092'
+buttonText: Call Pastor's Office
+image: safe ride.jpg
```
Comment (hawkrives) [inline data/contact-info/5-college-pastors.yaml:4]: Do we even use these images in the app?

**Review by hawkrives — APPROVED** [review]

**Comment by drewvolz** [comment]
> I'm starting to wonder if there is a need for a dedicated Resources section of the app, as important contacts seems to be a little cluttered (Pause Kitchen feels even more out of place now!).

Adding tabs is a decent stab at fixing the problem. I also would propose a `master > detail` listview so that more info can be found in a child view and the name/phone are listed up front... 

numbers-important

**Review by drewvolz — APPROVED** [review]

**Comment by hawkrives** [comment]
@drewvolz Those look nothing like a Master > Detail view 😝

**Comment by drewvolz** [comment]
Think of it as master > detail because we keep getting rid of the detail chevron

**Comment by hawkrives** [comment]
A plus icon doesn't make sense, either…

**Comment by drewvolz** [comment]
I'm not advocating for the + sign. I'm showing one example of a limited amount of info that we can get across, and then I am asking for you to imagine a detail view with more info.

**Comment by hawkrives** [comment]
pshhh imagination

**Pull request merged**